.github/ISSUE_TEMPLATE/packaging_tutorial.yml

```
@@ -0,0 +1,79 @@
+name: Packaging tutorial problem
+description: Request help for a problem following a packaging tutorial
+title: Trouble with a packaging tutorial
+
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thank you for providing feedback on Python packaging!
+
+      To help us help you, please fill out as much of the following as you can. If a question is not relevant, feel free to skip it.
+
+
+- type: input
+  id: os_version
+  attributes:
+    label: OS version
+    description: |
+      What is your operating system and version?
+
+      For example, this could be Windows 10 or Ubuntu 18.04.
+  validations:
+    required: true
+
+
+- type: input
+  id: py_version
+  attributes:
+    label: Python version
+    description: |
+      What is your Python version?
+
+      You can usually get this by running `python --version`.
+    placeholder: Python x.x.x
+  validations:
+    required: true
+
+
+- type: input
+  id: pip_version
+  attributes:
+    label: Pip version
+    description: |
+      What version of pip do you have?
+
+      You can usually get this by running `python -m pip --version`.
+    placeholder: pip x.x.x
+  validations:
+    required: true
+
+
+
+- type: input
+  id: guide
+  attributes:
+    label: Guide link
+    description: |
+      If you are following an online tutorial or guide, please provide a link to the page or section giving you trouble.
+
+
+- type: textarea
+  id: problem
+  attributes:
+    label: Problem description
+    description: |
+      Could you describe your issue in as much detail as possible?
+      
+      Please describe your problem, including all commands that you ran. Include any offline guides you followed, etc.
+  validations:
+    required: true
+
+
+- type: textarea
+  id: traceback
+  attributes:
+    label: Error message
+    description: |
+      If you received an error printout, please paste the _entire_ error message (called a traceback) here.
+    render: pytb
```

> There's no harm in trying to render random text as a traceback, and we can always remove it. It's best to target what is most common, and most commonly, this is a traceback. We can always removed the pytb from the issue if it doesn't apply and makes the text less appealing. But this should be more often right then not.

As a general philosophy, I've started leaning towards "most inclusive" over "most common". With that in mind, I'd really rather generalize this.

```suggestion
- type: textarea
  id: error
  attributes:
    label: Error message
    description: |
      If you received an error message, please copy and paste the _entire_ message here.
    render: text
```